README.md

```diff
@@ -85,6 +110,13 @@ To function correctly, this service principal must be assigned the [Contributor]
 The following steps describe how to create the service principal, assign the role, and create a secret in your repository with the resulting credentials.
 
 1. Open the Azure Cloud Shell at [https://shell.azure.com](https://shell.azure.com). You can alternately use the [Azure CLI](https://docs.microsoft.com/cli/azure/install-azure-cli?view=azure-cli-latest) if you've installed it locally. (For more information on Cloud Shell, see the [Cloud Shell Overview](https://docs.microsoft.com/azure/cloud-shell/overview).)
+
+    1.1 **(Required for Azure Stack Hub)** Run the following command to set the SQL Management endpoint to 'not supported'
```

Comment:
Please change this to "Required ONLY when **environment** is Azure Stack Hub" coz I assume this step is optional when its not Azure Stack.

Reply:
updated